src/styles/pages/search.scss

```diff
@@ -0,0 +1,31 @@
+@import "../vendors/include-media.scss";
+
+.product-listing {
```

Comment:
Maybe, we can find a way to extract this common block of style for both pages. 🤔

Reply:
Yeahh, I was thinking about that but I didn't find a good solution so far 🤔 
Suggestions?

Reply:
I paired with @renatamottam and we tried some solutions.  So far we extract the common block into a `ProductListing` component and reuse it on both pages Search and PLP.